src/firebase/remoteConfigValuesDefaults.ts

```diff
@@ -62,6 +62,7 @@ export const REMOTE_CONFIG_VALUES_DEFAULTS: Omit<
   celoWithdrawalEnabledInExchange: true,
   fiatConnectCashInEnabled: false,
   fiatConnectCashOutEnabled: false,
+  fiatConnectTimeoutSeconds: 30,
```

Comment:
Was considering 10s or 30s. Saw some mock-provider requests taking 5s, so went with 30s to be safe.

Reply:
yes 30s is good in a mobile environment.

Reply:
yes 30s is good in a mobile environment.